contracts/core/Core.sol

```diff
@@ -707,13 +707,15 @@ contract Core is MasterCopyNonUpgradable, ConsensusModule, MosaicVersion, CoreSt
      *
      * @param _validator A validator's address to join.
      *
-     * @return The total count of joined validators.
+     * @return validatorCount_ The total count of validators.
+     * @return minValidatorCount_ Minimum validator count required set by consensus.
+     * @return beginHeight_ Begin height of validator.
      */
     function joinBeforeOpen(address _validator)
         external
         onlyConsensus
         beforeOpen
-        returns(uint256 validatorCount_, uint256 minValidatorCount_)
+        returns(uint256 validatorCount_, uint256 minValidatorCount_, uint256 beginHeight_)
```

Comment:
```suggestion
        returns (uint256 validatorCount_, uint256 minValidatorCount_, uint256 beginHeight_)
```

Reply:
Updated. Also as it was exceeding the 79 char limit so updated the indentation